### PR TITLE
chore: Remove dead codes related to telemetry files.

### PR DESCRIPTION
**Pull request by JungukCho**
**Reason for Change**:

Remove dead codes from telemetry.go and telemetry_<suffix>.go files.

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

File: telemetry/telemetrybuffer.go [hunk 1]
```diff
@@ -327,17 +327,6 @@ func (tb *TelemetryBuffer) ConnectToTelemetryService(telemetryNumRetries, teleme
 	}
 }
 
-// TryToConnectToTelemetryService - Attempt to connect telemetry process without spawning it if it's not already running.
```
Comment (tamilmani1989) [inline telemetry/telemetrybuffer.go:330]: why was this removed?

Comment (JungukCho) [reply telemetry/telemetrybuffer.go:330]: @tamilmani1989  It was not called from any places. So, I think this is dead code. Do I miss something?

**Comment by JungukCho** [comment]
Close it to raise the another PR.
